lib/smart_answer/calculators/maternity_pay_calculator.rb

```diff
@@ -73,8 +73,12 @@ def initialize(due_date, leave_type = "maternity")
       @ssp_stop = @expected_week.weeks_after(-4).begins_on
     end
 
-    def self.payment_options(period)
-      PAYMENT_OPTIONS.fetch(period, {})
+    def self.payment_options
+      PAYMENT_OPTIONS
+    end
+
+    def payment_options
+      self.class.payment_options
     end
```

Comment:
Do we really need both of these methods?  Can it be done with just one?

Reply:
Within the flow definitionthe method is called directly within the `define` and `radio` calls where the a calculator instance doesn't exist. So it's easier to use a class method there.

Within the erb files, an instance of the calculator is easily accessible, so it's easier to use that there; that is to use the instance method.

Reply:
Going to resolve this following a conversation on the options here.